internal/v1/handler_compose_image.go

```diff
@@ -751,13 +751,12 @@ func (h *Handlers) buildCustomizations(ctx echo.Context, cr *ComposeRequest, d *
 	if cust.Users != nil {
 		var users []composer.User
 		for _, u := range *cust.Users {
-			groups := &[]string{"wheel"}
```

Comment:
I guess I'm stating the obvious here but I will do it anyway. Unless there is somewhere adding "wheel" to u.Group automatically this is a behavior change - I guess it's okay because the frontend is aware and takes care of this and we have no other consumers? Might still be nice to mention it in the commit message maybe?

Reply:
Fair point, I'll do that :)